psalm-baseline.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<files psalm-version="4.7.3@38c452ae584467e939d55377aaf83b5a26f19dd1">
+    <!-- NEXT_MAJOR: Remove the PHPCR issues -->
+    <file src="src/Admin/PHPCR/GalleryAdmin.php">
+        <InvalidExtendClass occurrences="1">
+            <code>BaseGalleryAdmin</code>
+        </InvalidExtendClass>
+        <MethodSignatureMismatch occurrences="1">
+            <code>protected function configureDatagridFilters(DatagridMapper $filter)</code>
+        </MethodSignatureMismatch>
+    </file>
+    <!-- NEXT_MAJOR: Remove this issue -->
```

Comment:
What's the issue ? How can you be sure all those can be remove in the next major ?

Reply:
Because I started this work on master, but I saw dev-kit requires all branches to be compatible with psalm, so I had to downgrade.

Reply:
The issue with those classes is that on those version of blockbundle and seo bundle the parent clases are marked with `@final`. On next seo-bundle / block-bundle this is solved because parent classes extend other clases that are not final.